packages/snaps-cli/src/cmds/build/bundleUtils.ts

```diff
@@ -128,3 +129,56 @@ export function postProcess(
 
   return processedString;
 }
+
+/**
+ * Processes dependencies and updates argv with an options object
+ * @param argv
+ */
+
+export function processDependencies(argv: YargsArgs) {
+  const { transpiledDeps, transpilationMode } = argv;
+  const options: Record<string, any> = {};
+  if (transpilationMode === TranspilationModes.localAndDeps) {
+    const regexpStr = getDependencyRegExp(transpiledDeps as string[]);
+    if (regexpStr !== null) {
+      options.ignore = regexpStr;
+    }
+  }
+  argv.options = options;
```

Comment:
The name `options` is too generic. In addition, `argv` can contain any number of different things that we may choose to call "options", and the risk of naming collisions is too high. We should rename it to `babelifyOptions`.